tests/jobs_test.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+from unittest.mock import MagicMock, patch
+
+from data_collections.jobs import getJobs
```

Comment:
Update the module path:
```suggestion
from data_collections.rss_parser import getJobs
```